projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -177,6 +184,25 @@ def get_value(self, key: str, default_value: str = None, is_required=True):
             )
         return value
 
+    def _get_atleast_one_value(self, key1: str, key2: str) -> Optional[str]:
+        value1 = self.get_value(key1, is_required=False)
+        value2 = self.get_value(key2, is_required=False)
+
+        if (not value1) and (not value2):
+            raise ValueError(
+                "Error occurred:\n"
+                f"What: Cannot configure heartbeat test\n"
+                f"Why: Missing required configuration with either key {key1} or key {key2}\n"
+                f"Consequences: The heartbeat will abort\n"
+                f"Countermeasures: Set as required configuration either as environment variable or in the config file."
+            )
+
+        return value1 or value2
```

Comment:
This would break if value is boolean . 

Perhaps return value1 if value1 is not None else value2
```suggestion
        if (value1 is None) and (value2 is None):
            raise ValueError(
                "Error occurred:\n"
                f"What: Cannot configure heartbeat test\n"
                f"Why: Missing required configuration with either key {key1} or key {key2}\n"
                f"Consequences: The heartbeat will abort\n"
                f"Countermeasures: Set as required configuration either as environment variable or in the config file."
            )

        return value1 if value1 is not None else value2
```

Reply:
Ooops, good catch. Fixed, thank you